android/src/main/java/com/novoda/notils/devicedetection/AmazonDetector.java

```diff
@@ -0,0 +1,17 @@
+package com.novoda.notils.devicedetection;
+
+class AmazonDetector {
+
+    //From: https://developer.amazon.com/appsandservices/solutions/devices/kindle-fire/specifications/01-device-and-feature-specifications
+    private static final String AMAZON = "Amazon";
+
+    private final String buildManufacturer;
+
+    AmazonDetector(String buildManufacturer) {
```

Comment:
would prefer if the `buildManufacturer` was the method param instead of ctor param. There's two cases this could be used:

- checking the device you're running on
- checking a list of buildManufacturers you get

For the first case, either way works, but for the second case, passing the `buildManufacturer` into the ctor means instantiating a new one each time you want to check.